lib/index.js

```diff
@@ -55,6 +56,55 @@ exports.plugin = {
             server.ext('onPreStart', internals.initialize);
             server.ext('onPostStop', internals.stop);
 
+            // Expose an hpal command for schwifty-migration
+            server.expose('commands', {
+                'migrate:diff': (initializedServer, args, rootDir, ctx) => {
```

Comment:
Should be an `async` function.

Reply:
Got a linting error cuz we don't have any awaits in the func

Reply:
Good point—we should `await` schwifty-migrate-diff's result!